src/arcade/core/sim/Series.java

```diff
@@ -30,6 +33,10 @@ public abstract class Series {
     /** Regular expression for fractions. */
     private static final String FRACTION_REGEX = "^(([0]*(\\.\\d*|))|(1[\\.0]*))$";
 
+    /** Regular expression for distributions. */
+    public static final String DISTRIBUTION_REGEX =
```

Comment:
Since we're implementing it this way, I sort of think it could be useful to also add a Bernoulli(p) distribution to things that are coin tosses (necro_frac, etc.) - if there a way to make this less specific to what is inside the parentheses and parse according to the type of distribution?

Reply:
I agree on adding a Bernoulli distribution. I can change the regex to accept one or two parameters in the distribution, which should capture most distributions we care about.